src/inc/corhost.h

```diff
@@ -389,6 +389,11 @@ class CorHost2 :
         LPCWSTR* argv,
         DWORD* pReturnValue);
 
+    static HRESULT PreloadAssembly(
+        LPCWSTR pwzAssemblyPath);
```

Comment:
A nit - could you please match the naming of other string arguments of methods in this class? That means naming this wszAssemblyPath.